dev/standalone-keycloak/docker-compose.yml

```diff
@@ -5,18 +5,21 @@ services:
   api:
     env_file:
       - './standalone-keycloak/galaxy_ng.env'
+      - '../.compose.env'
     environment:
       SOCIAL_AUTH_KEYCLOAK_PUBLIC_KEY: ${SOCIAL_AUTH_KEYCLOAK_PUBLIC_KEY}
       KEYCLOAK_HOST_LOOPBACK: ${KEYCLOAK_HOST_LOOPBACK}
```

Comment:
The assumption is that these values would come in from the `.compose.env` so you could probably remove each of this from the file below.

Reply:
@chambridge this other PR is changing it a little https://github.com/ansible/galaxy_ng/pull/945 the variable will need to be prefixed with `PULP_`

Reply:
ok.